crates/solvers/src/api/mod.rs

```diff
@@ -20,14 +22,18 @@ impl Api {
         shutdown: impl Future<Output = ()> + Send + 'static,
     ) -> Result<(), hyper::Error> {
         let app = axum::Router::new()
+            .layer(tower::ServiceBuilder::new().layer(
```

Comment:
Given the `DefaultBodyLimit::disable()` below, this is superfluous at the moment.